components/footer.tsx

```diff
@@ -6,7 +6,7 @@ export const Footer = () => {
   return (
     <div>
       <div className="font-poppins">
-        <div className={"mx-8  border-t-[1px] border-[#E4E4E4] pt-4"}>
+        <div className={"mx-8  border-t-[1px] border-[#707070] pt-4"}>
```

Comment:
we should define common colors in the tailwind theme

Reply:
Will do this later